meta-lmp-base/recipes-graphics/wayland/weston_%.bbappend

```diff
@@ -0,0 +1,6 @@
+# Weston with Xwayland support (requires X11 and Wayland)
+PACKAGECONFIG[xwayland] = "-Dxwayland=true,-Dxwayland=false,libxcursor"
+
+# Clients support
+SIMPLE_CLIENTS = "all"
+PACKAGECONFIG[clients] = "-Dsimple-clients=${SIMPLE_CLIENTS} -Ddemo-clients=true,-Dsimple-clients= -Ddemo-clients=false"
```

Comment:
From OE-core:

`PACKAGECONFIG[clients] = "-Dsimple-clients=all -Ddemo-clients=true,-Dsimple-clients= -Ddemo-clients=false"`

Why are we replicating this as well?

Reply:
This was a left over change from when `apalis-imx6` setup SIMPLE_CLIENTS differently (for some reason).  I'll drop this change as well.